node/buildpack.go

```diff
@@ -21,10 +21,12 @@ import (
 	"fmt"
 
 	"github.com/buildpack/libbuildpack/buildplan"
-	"github.com/heroku/libhkbuildpack/build"
-	"github.com/heroku/libhkbuildpack/detect"
+	"github.com/buildpack/libbuildpack/layers"
+	"github.com/buildpack/libbuildpack/logger"
 	"github.com/cloudfoundry/npm-cnb/modules"
```

Comment:
oh do we still need this cloudfoundry dependency?  should we fork the NPM buildpack or wait for the Heroku version that Lyra Langs is working on?

Reply:
I feel like `github.com/buildpack/libbuildpack/layers` should be `github.com/heroku/libhkbuildpack/layers`. We already have the fork and reference it elsewhere in this project. Maybe some tooling brought it back in?

Reply:
After looking around a bit, `github.com/buildpack/libbuildpack/layers` and `github.com/heroku/libhkbuildpack/layers` don't seem to have the same API. For one, the `hk` version expects a cache layer to exist. We're not using a cache at this point, so maybe this is OK for now. It's not like we're using `libcfbuildpack`, which is the more specifically pivotal one.

Reply:
> oh do we still need this cloudfoundry dependency? should we fork the NPM buildpack or wait for the Heroku version that Lyra Langs is working on?

It looks like we're just using the cloudfoundry deps for detection right now. I think it's fine to wait until we have the equivalent Heroku version, because I'm not sure we'd care to fork something just to change the name. I'm not opposed, though, it's just probably something for a different PR.